src/ru/skypro/ex5/ExerciseFive.java

```diff
@@ -0,0 +1,32 @@
+package ru.skypro.ex5;
+
+public class ExerciseFive {
+    public static void main(String [] args) {
+        int mashaSalary = 67_760;
+        int mashaSalaryYear = mashaSalary * 12;
+
+        float mashaNewSalary = mashaSalary * 1.1f;
+        float mashaNewSalaryYear = mashaNewSalary * 12;
+        float mashaDifference = mashaNewSalaryYear % mashaSalaryYear;
+
+        System.out.println("Маша теперь получает " + mashaNewSalary + " рублей. " + "Годовой доход вырос на " + mashaDifference + " рублей.");
+        int DenisSalary = 83_690;
```

Comment:
в этом случае также все переменные стоит именовать с маленькой бувы
```suggestion
        int denisSalary = 83_690;
```